collector/collector.go

```diff
@@ -691,6 +668,17 @@ func (c *collector) CollectMetrics(mts []plugin.Metric) ([]plugin.Metric, error)
 		return nil, fmt.Errorf("No metrics found")
 	}
 
+	// add labels as tags to metrics
+
+	for i := range metrics {
+		rid := metrics[i].Namespace[2].Value
+		if rid != "root" {
+			// add labels only for docker cointainer, skip the host
+			metrics[i].Tags = c.containers[rid].Specification.Labels
```

Comment:
I think it is better to add labels in sequence instead of override whole map with another